tests/parsers/moldev_softmax_pro/to_allotrope_test.py

```diff
@@ -53,17 +53,3 @@ def test_invalid_measurement_table_dimensions() -> None:
         match="Expected 4 rows in measurement table, got 3.",
     ):
         from_file(f"{TESTDATA}/errors/missing_measurement_table_row.txt", VENDOR_TYPE)
-
-
-@pytest.mark.parametrize(
-    "test_file",
-    [
-        f"{TESTDATA}/errors/lum_spectrum_columns.txt",
```

Comment:
It looks like this file still does not work with our parser.

Please add logic to detect this and raise an error message.

And example could be detecting if the read_type == Spectrum and the measurment type == lum, raising an error saying we don't support lum spectrum format at this time.